package.json

```diff
@@ -5,7 +5,7 @@
   "devDependencies": {
     "jasmine-core": "latest",
     "jshint": "latest",
-    "karma": "latest",
+    "karma": "^6.3.19",
```

Comment:
I think we want to push this change in a separate PR. Please create an issue and move that change there.

Reply:
https://github.com/radiasoft/sirepo/issues/4413